src/service/packet_router_service.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+
+package helium;
+
+import "packet_router.proto";
```

Comment:
so far we've been inlining the messages with the service. You mind inlining packet_router.proto?

Reply:
oh and run the formatter on it :-)

Reply:
Will do I keep it outside right now cause we don't have grpc implemented yet

Reply:
doesn't matter.. you can still use the messages defined in a service even if you're not done with the service yet

Reply:
it seems to not compile the `.hrl` file

Reply:
gah gpb madness... presumably @andymck has had this working just fine